frontend/src/components/Login.tsx

```diff
@@ -66,7 +67,7 @@ const Login = () => {
             setAuthInputs({ ...authInputs, email: event.target.value });
           }}
         />
-        <InputField
+        <PasswordField
           label="Password"
           type="password"
```

Comment:
We can remove the type prop of PasswordField and keep default type as "password" inside PasswordField.

`const [inputType, setInputType] = useState("password");`